{
    "CVE_data_meta": {
        "ASSIGNER": "secalert@redhat.com",
        "ID": "CVE-2013-4347",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "n/a",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "n/a"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "n/a"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "The (1) make_nonce, (2) generate_nonce, and (3) generate_verifier functions in SimpleGeo python-oauth2 uses weak random numbers to generate nonces, which makes it easier for remote attackers to guess the nonce via a brute force attack."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "n/a"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "name": "62388",
                "refsource": "BID",
                "url": "http://www.securityfocus.com/bid/62388"
            },
            {
                "name": "https://github.com/simplegeo/python-oauth2/issues/9",
                "refsource": "MISC",
                "url": "https://github.com/simplegeo/python-oauth2/issues/9"
            },
            {
                "name": "[oss-security] 20130912 Re: cve requests for python-oauth2",
                "refsource": "MLIST",
                "url": "http://www.openwall.com/lists/oss-security/2013/09/12/7"
            },
            {
                "name": "https://github.com/simplegeo/python-oauth2/pull/146",
                "refsource": "MISC",
                "url": "https://github.com/simplegeo/python-oauth2/pull/146"
            }
        ]
    }
}